scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -2208,3 +2208,35 @@ def foo2: Int =
       44
    end if
 end foo2
+<<< 2790 for loops indentation
+def update(tpf: Float): Unit =
+  val s = stateModifier(currentState, tpf)
+  currentState = s.state
+  processCommands(s.commands)
+  for (scene <- scenes.all) 
```

Comment:
can you add another loop with a single statement in the body? it looks like plain old scala2 code and should not use significant indentation. same with while.